logic.js

```diff
@@ -0,0 +1,93 @@
+
+const Tweeter = function(){
+  let posts = [ {
+      text: "First post!",
+      id: "p1",
+      comments: [
+          { id: "c1", text: "First comment on first post!" },
+          { id: "c2", text: "Second comment on first post!!" },
+          { id: "c3", text: "Third comment on first post!!!" }
+      ]
+  },
+  {
+      text: "Aw man, I wanted to be first",
+      id: "p2",
+      comments: [
+          { id: "c4", text: "Don't wory second poster, you'll be first one day." },
+          { id: "c5", text: "Yeah, believe in yourself!" },
+          { id: "c6", text: "Haha second place what a joke." }
+      ]
+  }]
+  let postCounter = 0
+  let postIdGenerator = 2
+  let commentIdGenerator = 6
+
+  //function for in-module use:
+  const getPost = function(postID,which){ // which: 0- return the post. 1- return post's index
```

Comment:
you did not need this function. The instructions let you know the functions you needed